components/create-project/index.js

```diff
@@ -81,6 +82,15 @@ class CreateProject extends Component {
           <SignInOrJoinFree />
         </Flex>
       );
+    } else if (parent?.isFrozen) {
+      return (
+        <Flex flexDirection="column" alignItems="center" my={6}>
+          <MessageBox withIcon type="warning">
+            <FormattedMessage defaultMessage="This account it frozen, you cannot create new projects at this time." />{' '}
+            <FormattedMessage defaultMessage="Please contact your fiscal host for more details." />
```

Comment:
Maybe link to the collective contact form; so users know how to contact the fiscal host? 🤔